### ISSUE TITLE
duplicate scala/annotation/nowarn.class with different file contents

### ISSUE THREAD

**Issue by jakeonfire** [scala/scala-collection-compat#520]
getting the following errors:
```
[error] 2 errors were encountered during merge
[error] java.lang.RuntimeException: deduplicate: different file contents found in the following:
[error] /root/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/modules/scala-collection-compat_2.12/2.4.4/scala-collection-compat_2.12-2.4.4.jar:scala/annotation/nowarn$.class
[error] /root/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.13/scala-library-2.12.13.jar:scala/annotation/nowarn$.class
[error] deduplicate: different file contents found in the following:
[error] /root/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/modules/scala-collection-compat_2.12/2.4.4/scala-collection-compat_2.12-2.4.4.jar:scala/annotation/nowarn.class
[error] /root/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.13/scala-library-2.12.13.jar:scala/annotation/nowarn.class
[error] 	at sbtassembly.Assembly$.applyStrategies(Assembly.scala:203)
[error] 	at sbtassembly.Assembly$.x$1$lzycompute$1(Assembly.scala:47)
[error] 	at sbtassembly.Assembly$.x$1$1(Assembly.scala:45)
[error] 	at sbtassembly.Assembly$.stratMapping$lzycompute$1(Assembly.scala:45)
[error] 	at sbtassembly.Assembly$.stratMapping$1(Assembly.scala:45)
[error] 	at sbtassembly.Assembly$.inputs$lzycompute$1(Assembly.scala:128)
[error] 	at sbtassembly.Assembly$.inputs$1(Assembly.scala:118)
[error] 	at sbtassembly.Assembly$.apply(Assembly.scala:145)
[error] 	at sbtassembly.Assembly$.$anonfun$assemblyTask$1(Assembly.scala:320)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error] 	at sbt.Execute.work(Execute.scala:291)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[error] 	at java.base/java.lang.Thread.run(Thread.java:835)
[error] (common / assembly) deduplicate: different file contents found in the following:
[error] /root/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/modules/scala-collection-compat_2.12/2.4.4/scala-collection-compat_2.12-2.4.4.jar:scala/annotation/nowarn$.class
[error] /root/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.13/scala-library-2.12.13.jar:scala/annotation/nowarn$.class
[error] deduplicate: different file contents found in the following:
[error] /root/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/modules/scala-collection-compat_2.12/2.4.4/scala-collection-compat_2.12-2.4.4.jar:scala/annotation/nowarn.class
[error] /root/.cache/coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.13/scala-library-2.12.13.jar:scala/annotation/nowarn.class
```
but i thought this was [fixed](https://github.com/scala/scala-collection-compat/pull/417/commits/86475b4cb540f81723d6b09753355aff75c149e3) in 2.4.0... or am i misinterpreting that commit?

**Comment by jakeonfire** [comment]
working around it with
```sbt
    case PathList("scala", "annotation", "nowarn.class") => MergeStrategy.last
    case PathList("scala", "annotation", "nowarn$.class") => MergeStrategy.last
```
but seems like this shouldn't be necessary

**Comment by SethTisue** [comment]
Is the problem also reproducible on Scala 2.12.15?

**Comment by SethTisue** [comment]
This was fixed by #460. You are using an outdated version scala-collection-compat; try 2.6.0.
